package.json

```diff
@@ -41,7 +41,11 @@
         {
           "SwitchCase": 1
         }
-      ]
+      ],
+      "prefer-const": ["error", {
+        "destructuring": "any",
+        "ignoreReadBeforeAssign": false
+      }]
```

Comment:
whitespace formatting. also, use tabs in this file?

Reply:
oh. hadn't thought about it. My package.json is all spaces. Is it different on your end because this may be my editor changing things up.

Reply:
Yes, to be clear, I was requesting you follow the existing whitespace formatting in the file.

I was only suggesting to use tabs in the file, instead of spaces.

Reply:
ah. can do.